src/reanimated2/js-reanimated/index.ts

```diff
@@ -112,8 +113,10 @@ export const _updatePropsJS = (
         component._touchableNode.setAttribute(dashedKey, rawStyles[key]);
       });
     } else {
+      const componentName =
+        'className' in component ? component?.className : '';
       console.warn(
-        `[Reanimated] It's not possible to manipulate the component ${component}`
+        `[Reanimated] It's not possible to manipulate the component ${componentName}`
```

Comment:
Is it actually what we want here?

Reply:
Well, I've tested the following code:
```tsx
          const object = { a: 6, b: 7 };
          console.log(`1. I've got an object: ${object}`);
          console.log(`2. I've got an object: ${object.toString()}`);
```

It prints:  
1. I've got an object: [object Object]
2. I've got an object: [object Object]

I can't reproduce this particular error (it should be unaccessible), but we definitely don't want to print "[object Object]" and className is the best we can extract from the component.

Reply:
Yeah but in web it might be different. Just try logging this string in the beginning of this function and see if it makes sense.

Reply:
Tested on web too, still the same log